web-admin/src/routes/+page.ts

```diff
@@ -0,0 +1,20 @@
+import { redirect } from "@sveltejs/kit";
+import {
+  adminServiceFindOrganizations,
+  adminServiceGetCurrentUser,
+} from "../client";
+import { ADMIN_URL } from "../client/http-client";
+
+// The current user is not available on the web server, so SSR would screw up the redirect logic
+export const ssr = false;
+
+export async function load() {
+  const user = await adminServiceGetCurrentUser();
+
+  if (user.user) {
+    const orgs = await adminServiceFindOrganizations();
+    throw redirect(307, `/${orgs.organization[0].name}`);
+  } else {
+    throw redirect(307, `${ADMIN_URL}/auth/login?redirect=${window.origin}`);
+  }
+}
```

Comment:
1. Could this be done in the page Svelte component using svelte-query? Then the page might load faster and it could show a loading spinner while looking up the user and project?
2. On that note, does the `redirect` only trigger 307 for the external URL? For the org page navigation, it should just navigate within the SPA (no page browser refresh)
3. Also, `orgs` might be empty (we won't add a default org), so it should show a friendly "you're lonely" message in that case

Reply:
(If there's anything to do for bullets 1 and 2 above, we could do it in a different PR and merge this to get the experience deployed to staging asap)

Reply:
I'll address 1 and 2 after lunch – can do in this PR or another.